src/Data/ObfPoiSectionReader_P.cpp

```diff
@@ -22,6 +22,10 @@
 #include <Logging.h>
 
 const int BUCKET_SEARCH_BY_NAME = 5;
+const int BASE_POI_SHIFT = 7;
+const int PRECISED_POI_SHIFT = 5;
```

Comment:
PRECISED_POI_SHIFT rename to FINAL_POI_SHIFT, PRECISED_POI_ZOOM = FINAL_POI_ZOOM. (By the way, it's precise not precised)